RecoVertex/AdaptiveVertexFinder/interface/TrackVertexArbitration.h

```diff
@@ -187,12 +204,18 @@ std::vector<VTX> TrackVertexArbitration<VTX>::trackVertexArbitrator(
 		Measurement1D isv = dist.distance(VertexState(RecoVertex::convertPos(sv->position()),RecoVertex::convertError(sv->error())),VertexState(refPoint, refPointErr));	   
 
 		float dR = Geom::deltaR2(flightDir,tt.track()); //.eta(), flightDir.phi(), tt.track().eta(), tt.track().phi());
+		
+		double timeSig = 0.;
+		if( svHasTime && edm::isFinite(tt.timeExt()) ) {
+		  double tError = std::sqrt( std::pow( tt.dtErrorExt(), 2 ) + svTimeCov );
+		  timeSig = std::abs(tt.timeExt() - svTime)/tError;
```

Comment:
may want to check for divide by zero here...

Reply:
this is again ensured by code elsewhere.